specification/Overview.md

```diff
@@ -9,18 +9,53 @@ organised by data category (saveframe).
 
 ### General issues
 
-  1. Residue and Atom names
-
-    Residues are identified by three strings ('chain_code', 'sequence_code',
-    'residue_type') that always are used together. Atoms are identified by a
-    residue identifier plus the atom name as a fourth string.
+  1. Requirements for NEF compatibility
+
+    The NEF format tries to support as many kinds of data as possible in a
+    simple file format. But we can clearly not require any program to
+    read and understand data that it does not normally support, be it complex or
+    crosslinked sequences, non-standard residues or residue variants, OR/AND
+    restraint logic, anonymous atom assignments, particular restraint types,
+    etc. If a program does support a certain type of data,
+    it is required to read and write such data correctly to/from NEF. If it
+    does not, it is free to skip such data, or convert them to something it can
+    deal with if this can be done appropriately.
```

Comment:
>  it is required to read and write such data correctly to/from NEF. If it does not, it is free to skip such data, or convert them to something it can deal with if this can be done appropriately.

Data exchange is the main purpose of NEF. It is not clear to me what does 'fee to skip such data'  mean. The correct approach would be the data should be written out as it is in the output NEF file.  Why I am telling this is the following. Suppose softare A writes an output NEF file, which contains some data which can be understood only by software A . This file is given to software B as an input and B produces an output NEF file. Then the user decides that this data has to go through software A again. Software B might have removed some of the data relevant(but critical)  to only software A .I hope you understand the situation. This is not very uncommon in NMR structure calculation.  So its better to modify the statement like if the data can' t be parsed or understood by the software, then the data should be presented in the output NEF file as it is.  This will ensure lossless data exchange.